locksmith/src/sequelize.js

```diff
@@ -0,0 +1,4 @@
+const models = require('../models')
```

Comment:
This seems the wrong name for this file. sequelizeLock.js? Also why is it needed? Used in several places this way?

Reply:
Good point, initially that was written with the idea that multiple models could be loaded from this. I will rename this to better reflect the scenario that exists right now; we have a single model